source/agora/node/FullNode.d

```diff
@@ -466,4 +486,69 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    public void pushBlock (const ref Block block) @safe
+    {
+        foreach (BlockExternalizedHandler handler; block_handlers)
+        {
+            runTask({
+                try
+                {
+                    // API POST: /block_externalized
+                    handler.pushBlock(block);
+                }
+                catch (Exception e)
+                {
+                    log.error(
+                        "Error sending block height #", block.header.height
```

Comment:
Will this actually work? I didn't realize it was supported. I thought we had to use `log.error("block height #{}", height)` - the formatting string.